### PR TITLE
Fixes after updating of the guide.

### PR DESCRIPTION
**Pull request by EndrII**
This PR contains fixes after last review in the #537 PR

### CODE REVIEW

File: docs/en/Guide.md [hunk 1]
```diff
@@ -97,13 +97,13 @@ cqtdeployer -bin myexecutable -libDir /PathToMyExtraLibs -recursiveDepth 5 -qmlD
 # Using a live animation example from [Qt Examples](https://doc.qt.io/qt-5/qtquick-animation-example.html) 
 
 1. Build the project as a release
-    1. Run qamke with the -r option for release build. 
+    1. Run qmake with the -r option for release build.
```
Comment (pzhlkj6612) [inline docs/en/Guide.md:100]:
```suggestion
    1. Run qmake with the -r option for release build
```

File: docs/en/Guide.md [hunk 2]
```diff
@@ -564,7 +561,7 @@ QMAKE_EXTRA_TARGETS += deploy
 2. Open the project control panel 
     ![Снимок экрана от 2021-02-22 12-15-51](https://user-images.githubusercontent.com/12465465/108687472-d09b6d80-7507-11eb-9b56-8d09d42dac5e.png)
 
-3. Choosing a release build.
+3. Choose a release build.
```
Comment (pzhlkj6612) [inline docs/en/Guide.md:564]:
```suggestion
3. Choose a release build
```

File: docs/en/Guide.md [hunk 3]
```diff
@@ -575,6 +572,6 @@ QMAKE_EXTRA_TARGETS += deploy
     ![изображение](https://user-images.githubusercontent.com/12465465/108687800-35ef5e80-7508-11eb-8f69-aec090d4baa0.png)
 
 5. Save
-6. Start the build. 
+6. Start the building.
```
Comment (pzhlkj6612) [inline docs/en/Guide.md:575]:
```suggestion
6. Start the building
```

**Review by pzhlkj6612 — CHANGES_REQUESTED** [review]
The period (.) at the end of the ordered list item is inconsistent.

**Comment by EndrII** [comment]
@pzhlkj6612 done

**Review by pzhlkj6612 — APPROVED** [review]
OK.

**Pull request merged**